crates/ruff_linter/src/rules/flake8_bugbear/rules/getattr_with_constant.rs

```diff
@@ -65,6 +66,13 @@ pub(crate) fn getattr_with_constant(checker: &Checker, expr: &Expr, func: &Expr,
     if is_mangled_private(value.to_str()) {
         return;
     }
+    // Ignore non-NFKC attribute names. Python normalizes identifiers using NFKC, so using
+    // attribute syntax (e.g., `obj.attr`) would normalize the name and potentially change
+    // program behavior.
+    let attr_name = value.to_str();
+    if attr_name.nfkc().collect::<String>() != attr_name {
+        return;
+    }
```

Comment:
Let's move this after line 78 as this comparison is more expensive than testing if it is the builtin getattr